example/Jamfile

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2022 Alan de Freitas (alandefreitas@gmail.com)
+#
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+#
+# Official repository: https://github.com/CPPAlliance/url
+#
+
+build-project grammar ;
```

Comment:
What is the "grammar" project?

Reply:
Just the targets in `example/grammar`. Should this "`add_subdirectory`" be expressed differently?

Reply:
I think "grammar" should be renamed to "magnet"